### PR TITLE
Renamings, cleanups and other updates

### PR DESCRIPTION
**Pull request by LorbusChris**
/cc @lmzuccarelli
/cc @sherine-k

### CODE REVIEW

**Comment by lmzuccarelli** [comment]
@LorbusChris - nice cleanup !!

I didn't get a chance to remove the old build pipeline  - could you do that in this PR and also update the README with the latest tree

File: environments/overlays/local/pipelineruns/okd-coreos-all-pipelinerun.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: okd-coreos-all-pipelinerun-
+spec:
+  params:
+    - name: repo
```
Comment (lmzuccarelli) [inline environments/overlays/local/pipelineruns/okd-coreos-all-pipelinerun.yaml:7]: Excellent - passing these as parameters !!!

File: manifests/tekton/tasks/base/cosa-build.yaml [hunk 2]
```diff
@@ -20,16 +15,13 @@ spec:
...
-        echo $(params.version)
-
-        cd /srv/scos
-
+        cd /workspace/coreos
         cosa fetch
-
-        cosa build
+        cosa build ostree
+        cosa buildextend-extensions
```
Comment (sherine-k) [inline manifests/tekton/tasks/base/cosa-build.yaml:23]: Should we add the `COSA_NO_KVM=1` here? or maybe next PR, once we narrow down the root cause for the slowness

**Review by lmzuccarelli — COMMENTED** [review]
Great work Christian.

I will verify the changes locally

File: manifests/tekton/pipelines/base/okd-coreos-all.yaml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    - name: cosa-test
+      runAfter:
+        - cosa-build
+      taskRef:
+        kind: Task
+        name: cosa-test
+      workspaces:
+        - name: ws
+          workspace: shared-workspace
+    - name: cosa-buildextend
```
Comment (sherine-k) [inline manifests/tekton/pipelines/base/okd-coreos-all.yaml:69]: Hi @LorbusChris 
Just for me to understand... I thought this buildextend was part of the scos , and needed to be done before the `cosa-test`

**Comment by LorbusChris** [comment]
> I didn't get a chance to remove the old build pipeline - could you do that in this PR and also update the README with the latest tree

That should be in here already :slightly_smiling_face: also passed locally in a kind cluster.
Splitting it like this hopefully makes it a bit easier to tell where COSA barfs, and why

**Review by sherine-k — APPROVED** [review]
Thanks @LorbusChris 
I just added a comment for the COSA_NO_KVM=1 , in case you want to add it!

File: manifests/tekton/pipelines/base/okd-coreos-all.yaml (hunk 3 above)
Comment (LorbusChris) [reply manifests/tekton/pipelines/base/okd-coreos-all.yaml:69]: the qemu test only needs buildextend-qemu and only does baseline testing - it makes sense to gate other builtextends on passing of that qemu test (and then afterwards run their specific tests also, but we don't do that yet)

File: manifests/tekton/tasks/base/cosa-build.yaml (hunk 2 above)
Comment (LorbusChris) [reply manifests/tekton/tasks/base/cosa-build.yaml:23]: great point. I think we should first find out where exactly it'll be needed.

File: manifests/tekton/pipelines/base/okd-coreos-all.yaml (hunk 3 above)
Comment (lmzuccarelli) [reply manifests/tekton/pipelines/base/okd-coreos-all.yaml:69]: @LorbusChris - great approach, makes the process easier and allows more flexibility

**Comment by lmzuccarelli** [comment]
Verified locally - Good to merge

```
tkn pr describe okd-coreos-pipelinerun-all -n okd-coreos-pipeline
Name:              okd-coreos-pipelinerun-all
Namespace:         okd-coreos-pipeline
Pipeline Ref:      okd-coreos-all
Service Account:   default
Labels:
 tekton.dev/pipeline=okd-coreos-all

🌡️  Status

STARTED          DURATION   STATUS
31 minutes ago   30m27s     Succeeded

⚓ Params

 NAME        VALUE
 ∙ version   4.12

📂 Workspaces

 NAME                 SUB PATH   WORKSPACE BINDING
 ∙ shared-workspace   ---        VolumeClaimTemplate

🗂  Taskruns

 NAME                                              TASK NAME            STARTED          DURATION   STATUS
 ∙ okd-coreos-pipelinerun-all-cosa-buildextend     cosa-buildextend     9 minutes ago    8m24s      Succeeded
 ∙ okd-coreos-pipelinerun-all-cosa-test            cosa-test            12 minutes ago   3m5s       Succeeded
 ∙ okd-coreos-pipelinerun-all-cosa-build           cosa-build           24 minutes ago   12m19s     Succeeded
 ∙ okd-coreos-pipelinerun-all-cosa-init            cosa-init            28 minutes ago   3m40s      Succeeded
 ∙ okd-coreos-pipelinerun-all-rpm-artifacts-copy   rpm-artifacts-copy   31 minutes ago   2m59s      Succeeded
```

**Pull request merged**